councilmatic/urls.py

```diff
@@ -129,9 +132,12 @@
         name="lametro_ses_endpoint",
     ),
     path("smartlogic/", include("smartlogic.urls", namespace="smartlogic")),
+    path("cms/", include(wagtailadmin_urls)),
+    path("documents/", include(wagtaildocs_urls)),
+    path("", include(wagtail_urls)),
```

Comment:
Ordering it this way means the Wagtail page will take precedence as we move additional pages over to the CMS.